Tasks/DeployAzureResourceGroup/AzureUtilityRest.ps1

```diff
@@ -23,7 +23,7 @@ function Create-AzureResourceGroupIfNotExist
         if(-not $azureResourceGroup -and -not [string]::IsNullOrEmpty($location))
```

Comment:
Add a verbose statement when ignoring error while verifying the existence of resource group earlier in this function